ors-api/src/test/java/org/heigit/ors/apitests/snapping/ParamsTest.java

```diff
@@ -83,7 +83,7 @@ private static JSONArray createLocations(JSONArray... locations) {
     private static JSONObject validBody() {
         return new JSONObject()
                 .put("locations", createLocations(location94m(), location2m()))
```

Comment:
😂  what about the whole don't repeat strings thing in the last PR

Reply:
Fixed. There were a few more.